### ISSUE TITLE
roq videos bug in quake3

### ISSUE THREAD

**Issue by Alex3474247** [glKarin/com.n0n3m4.diii4a#237]
I'm having this instead of roq videos in quake3.
https://drive.google.com/file/d/1A8hUMJHPSiirxLxXPxuehr9QsB3p_aQ_/view?usp=drivesdk
Can you fix it?
The gameplay is ok but videos.
![Screenshot_20240728_220038_com karin idTech4Amm](https://github.com/user-attachments/assets/30df4594-d48f-458f-83a2-8ebef4d20d30)
[stderr.txt](https://github.com/user-attachments/files/16404942/stderr.txt)
[stdout.txt](https://github.com/user-attachments/files/16404943/stdout.txt)

**Comment by lvonasek** [comment]
I had a similar issue in another ioquake3 port. I fixed that this way:
https://github.com/DrBeef/ioq3quest/pull/1

**Comment by Alex3474247** [comment]
No, unfortunately it didn't help when I'm in single player mode and reset my single player to play from the beginning. If select cinematics from the quake3 menu, it's ok.
